C4SGWeb/src/app/project/list/project-list.component.css

```diff
@@ -1,54 +1,85 @@
-.selected {
-    background-color: #CFD8DC !important;
-    color: white;
-}
-.projects {
-    margin: 0 0 2em 0;
-    list-style-type: none;
-    padding: 0;
-    width: 70em;
-}
-.projects li {
-    cursor: pointer;
-    position: relative;
-    left: 0;
-    background-color: #EEE;
-    margin: .5em;
-    padding: .3em 0;
-    height: 15em;
-    border-radius: 4px;
-}
-.projects li:hover {
-    color: #607D8B;
-    background-color: #DDD;
-    left: .1em;
-}
-.projects li.selected:hover {
-    background-color: #BBD8DC !important;
-    color: white;
-}
-.projects .text {
-    position: relative;
-    top: -3px;
+.container .row {
+  padding-top: 1rem;
 }
-
-button {
-    font-family: Arial;
-    background-color: #eee;
-    border: none;
-    padding: 5px 10px;
-    border-radius: 4px;
-    cursor: pointer;
-    cursor: hand;
-}
-button:hover {
-    background-color: #cfd8dc;
-}
-button.delete {
+.form-block {
+  padding-right: 20px;
+}
+form {
+  padding: 1em 1.25em;
+  background-color: #EEE;
+  border-radius: 4px;
+}
+.form-block .input-field {
+  max-width: 300px;
+  margin: 0 auto;
+  float: none;
+}
+.form-block button {
+  margin-top: 0.4rem;
+}
+.projects-block {
+  margin: 0 0 2em 0;
+  list-style-type: none;
+}
+.projects-block li {
+  display: block;
+  cursor: pointer;
+  position: relative;
+  left: 0;
+  padding: 1em 0.5em;
+  background-color: #EEE;
+  border-radius: 4px;
+}
+.projects-block li:hover {
+  color: #607D8B;
+  background-color: #DDD;
+  left: .1em;
+}
+.projects-block .selected {
+  background-color: #CFD8DC !important;
+  color: white;
+}
+.projects-block li.selected:hover {
+  background-color: #BBD8DC !important;
+  color: white;
+}
+.card-content {
+  margin-top: 0.5rem;
+}
+.card-content a{
+  display: block;
+}
+.card-content h5, h6 {
+  display: inline-block;
+  text-decoration: underline;
+  margin-top: 0;
+}
+.card-content h6 {
+  margin-bottom: 0;
+}
+.card-content-left-side.col, .card-content-right-side.col, .card-content-description {
+  padding-left: 0;
+}
+.card-content-right-side.col {
+  padding-right: 0;
+}
+.card-content-right-side span {
+  display: block;
+}
+.card-content-right-side span:first-child {
+  margin: 0.5rem 0 0.656rem;
+}
+@media screen and (min-width:993px)  {
+  .container {
+    width: 90%;
+  }
```

Comment:
Well.. Since .container is part of the framework... We could have some side effects. I am not sure which ones. Or perhaps, no one.
Take a look at : http://materializecss.com/grid.html